src/Microsoft.AspNetCore.Mvc/Microsoft.AspNetCore.Mvc.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Microsoft.AspNetCore.Mvc.Analyzers\Microsoft.AspNetCore.Mvc.Analyzers.csproj" PrivateAssets="None" />
```

Comment:
This is a nuspec dependency only, right? If so, you should add `ReferenceOutputAssembly=false`.